eng/build.ps1

```diff
@@ -134,6 +134,11 @@ foreach ($argument in $PSBoundParameters.Keys)
   }
 }
 
+# Default the BuildTargetFramework to the latest netcorapp version
+if (!$framwork) {
```

Comment:
$framwork -> $framework?

Reply:
Also, this should instead be done in MSBuild right? I thought we already did that.
https://github.com/dotnet/runtime/blob/master/src/libraries/Directory.Build.props#L21

Reply:
It needs to be set in the build entry scripts now to differentiate between a root build and an individual project build which now builds all configurations by default.